crates/rustc_codegen_spirv/src/linker/capability_computation.rs

```diff
@@ -104,28 +72,36 @@ fn operand_required_extensions(op: &Operand) -> &'static [&'static str] {
 // It says no. We strip it. Things explode.
 // So, this function is to encode any special version-specific rules that aren't in rspirv.
 fn additional_extensions(module: &Module, inst: &Instruction) -> &'static [&'static str] {
-    if module.header.as_ref().unwrap().version() < (1, 5)
-        && inst.class.opcode == Op::Capability
-        && inst.operands[0].unwrap_capability() == Capability::VulkanMemoryModel
-    {
-        &["SPV_KHR_vulkan_memory_model"]
+    let version = module.header.as_ref().unwrap().version();
```

Comment:
Probably good to move this let inside the if statement, as this is really hot code